### DISCUSSION TITLE
How to enable sysmon?

### DISCUSSION THREAD

**Discussion by gemini2186** [kingToolbox/WindTerm#1183] [General]
The feature is described here https://kingtoolbox.github.io/2022/08/21/sysmon , but how to enable it?
I do not have such a button (ssh, connect to rhel)
In MobaXterm it works without problems.
ty!

**Comment by shaojs321** [comment]
最新的预览版[WindTerm 2.6.0 Prerelease 1]才有这功能，不过不太稳定，你可以试试
